.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -1,13 +1,12 @@
 name: Bug report
-description: Bug reports require a minimal reproduction and should be preceded by a Discussion first.
-labels: ['type:bug', 'status:requirements', 'priority-5-triage']
+description: Bug reports are only for maintainers. If you are a Renovate user, create a Discussion instead.
+labels: ['type:bug', 'priority-5-triage', 'needs-discussion']
```

```suggestion
labels: ['type:bug', 'status:requirements', 'priority-5-triage', 'needs-discussion']
```

I think we should keep a default `status:` type label.